app/uk/gov/hmrc/nationaldutyrepaymentcenter/models/UploadedFile.scala

```diff
@@ -42,7 +42,8 @@ case class UploadedFile(
   uploadTimestamp: ZonedDateTime,
   checksum: String,
   fileName: String,
-  fileMimeType: String
+  fileMimeType: String,
+  fileSize: Option[Int] = None
```

Comment:
I added this when I thought we had that data from the FE.   We don't (yet?) but I've kept it in so that if we start collecting file size from upscan response in the FE and propagate it to the BE it will automatically get added to the file transfer request and audit events.